xdsl/irdl.py

```diff
@@ -1101,7 +1101,7 @@ def builder(
                                successors, regions)
 
     new_attrs["build"] = classmethod(builder)
-    new_attrs["irdl_definition"] = classmethod(property(lambda cls: op_def))
```

Comment:
pyright never liked this, for some reason it's ok with `@classmethod @property` but not `classmethod(property`

Reply:
Is that the only reason you are removing this feature?
What is the error you get? Can we just `#type: ignore` it if this doesn't work for wrong reasons?